README.md

```diff
@@ -161,6 +161,14 @@ Tell the player to show a placeholder that indicates an error.
 
 Example: `playerApi.error(new Error('Something bad happened to my gadget'))`
 
+#### `recordActivity`
```

Comment:
Any constraints will be handled by the player. But I think your point is no less valid... I'm going to keep this out of documentation for now while it's not necessarily a public API.